### PR TITLE
v2.0.0+galaxy0

### PR DESCRIPTION
**Pull request by RJMW**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -109,9 +109,9 @@ script:
   - set -e
   - |
     if [ -s changed_tools_chunk.list ]; then
-        planemo test --galaxy_branch "$GALAXY_RELEASE" --galaxy_source "$GALAXY_REPO" $(cat changed_tools_chunk.list)
+        planemo test --galaxy_branch "$GALAXY_RELEASE" --galaxy_source "$GALAXY_REPO" --conda_ensure_channels computational-metabolomics,iuc,bioconda,conda-forge,defaults $(cat changed_tools_chunk.list)
```
Comment (bgruening) [inline .travis.yml]: is compuational_metabl still needed?

File: tools/dimspy/merge_peaklists.xml [hunk 2]
```diff
@@ -10,52 +10,60 @@
...
+                                                       e.g. [pl1 pl2], [pl3, pl4] can be converted to 2 new lists:
                                                             [pl1, pl3] [pl2, pl4]"/>
-            <when value="true">
-                <param name="filelist" argument="--filelist" type="data" format="tsv,tabular" label="Filelist" />
-            </when>
-            <when value="false">
-                <param name="filelist" argument="--filelist" type="data" format="tsv,tabular" optional="true" label="Filelist" />
-            </when>
-        </conditional>
+        <param name="filelist" type="data" format="tsv,tabular" optional="true" label="Filelist" argument="--filelist"  />
```
Comment (bgruening) [inline tools/dimspy/merge_peaklists.xml]: if argumen is --filelist you can remove the `name` I would also add some more help to this parameter I guess

File: .travis.yml (hunk 1 above)
Comment (RJMW) [reply .travis.yml]: I have removed the ```--conda_ensure_channels```parameter

File: tools/dimspy/merge_peaklists.xml (hunk 2 above)
Comment (RJMW) [reply tools/dimspy/merge_peaklists.xml]: I have kept the name for consistency with other params (e.g. hdf5_files_in and multilist). I have added a short help section.

**Review by Tomnl — APPROVED** [review]

**Pull request merged**